src/test/java/uk/gov/hmcts/reform/civil/helpers/hearingsmappings/HearingsPartyMapperTest.java

```diff
@@ -981,6 +981,122 @@ void shouldBuildPartyDetails_whenClaimantResponds1v2SS() {
         assertThat(actualPartyDetailsModel).isEqualTo(expected);
     }
 
+    @Test
+    void shouldBuildPartyDetails_whenClaimantResponds1v1_nullOrg() {
+        CaseData caseData = CaseDataBuilder.builder()
+            .addApplicant1LitigationFriend()
+            .addRespondent1LitigationFriend()
+            .atStateApplicantRespondToDefenceAndProceed()
+            .addApplicant1ExpertsAndWitnesses()
+            .addRespondent1ExpertsAndWitnesses()
+            .build().toBuilder()
+            .applicant1OrganisationPolicy(uk.gov.hmcts.reform.ccd.model.OrganisationPolicy.builder()
+                                              .orgPolicyCaseAssignedRole("[APPLICANTSOLICITORONE]").build()).build();
+
+        PartyDetailsModel applicantPartyDetails = buildExpectedIndividualPartyDetails(
```

Comment:
for my code taste.
I like to make the test easy to read. 
My suggestion is... We can extract method for all these lines .

something like this.
assertThat(actualPartyDetailsModel).isEqualTo(getExpected());

What do you think?